gatherers/offline.js

```diff
@@ -86,6 +88,15 @@ class Offline extends Gather {
       expression: requestPage
     }).then(_ => {
       return Offline.pollForOfflineResponseStatus(driver, 0);
+    }, _ => {
+      // Account for execution errors.
+      return {
+        result: {
```

Comment:
It's unclear that this will be treated as an error or an edge case by auditors.

Should the description, value, type or something else highlight that this is an error case?